### ISSUE TITLE
Flag: html in header

### ISSUE THREAD

**Issue by steveklabnik** [steveklabnik/rustdoc#144]
This issue is part of https://github.com/steveklabnik/rustdoc/issues/125

## `--html-in-header`: include more HTML in <head>

Using this flag looks like this:

```bash
$ rustdoc src/lib.rs --html-in-header header.html
$ rustdoc README.md --html-in-header header.html
```

This flag takes a list of files, and inserts them into the `<head>` section of
the rendered documentation.

----------------------------------------------------

Should we keep this flag or not?

**Comment by steveklabnik** [comment]
I think we should support this use-case, but I'm not sure that this flag is the right way to do it.

**Comment by mgattozzi** [comment]
A lot of these seem to be ways to extend or change how rustdoc looks. Instead of a feature flag maybe we can just read from a directory that will overide how the docs look for it.

**Comment by steveklabnik** [comment]
Yeah, I'd be down for that, or maybe something like how mdbook does it

http://azerupi.github.io/mdBook/format/theme/theme.html

**Comment by mgattozzi** [comment]
That seems like a good idea. Just having a theme directory to automatically theme it all seems solid.

**Comment by steveklabnik** [comment]
We talked about this at the tools team meeting today and the intention is to figure out support for lightly customizing rustdoc's output as a first-class thing, but not necessarily via this low-level mechanism.
